Persistence/Contexts/AppDbContext.cs

```diff
@@ -33,6 +54,17 @@ protected override void OnModelCreating(ModelBuilder builder)
             builder.Entity<Release>().Property(r => r.Date).HasDefaultValue(DateTime.UtcNow);
             builder.Entity<ReleaseReleaseNote>().HasKey(
                 rrn => new {rrn.ReleaseId, rrn.ReleaseNoteId});
+            
+            // All entities that require extra security for isPublic
+            builder.Entity<Release>().HasQueryFilter(
+                r => r.IsPublic && !UserIsAdmin || !r.IsPublic && UserIsAdmin ||
```

Comment:
kan vel bli forenkla til `!(!r.isPublic && !UserIsAdmin)` (sjekke kun siste entry i truth tabellen i staden for dei tre første)

Reply:
Ja

Reply:
c220d19